generators/client/templates/react/package.json.ejs

```diff
@@ -75,7 +75,7 @@ limitations under the License.
     "@types/node": "10.14.3",
     "@types/react": "16.8.8",
     "@types/react-dom": "16.8.3",
-    "@types/react-redux": "7.0.5",
+    "@types/react-redux": "6.0.6",
```

Comment:
Provably you can upgrade to 6.0.13

Reply:
Thanks @DanielFran. I did upgrade it. Lets see the CI results as I did not test it locally but it should work out of the box I guess

Reply:
@DanielFran That did not work either. Logs: https://jhipster.visualstudio.com/generator-jhipster/_build/results?buildId=2671. It seems there code changes needed for 6.0.13 as well